src/loaders/compute_tools/checkm2/checkm2.py

```diff
@@ -68,6 +69,13 @@ def _run_checkm2(
         fatal_tuples.append(fatal_tuple)
     write_fatal_tuples_to_dict(fatal_tuples, output_dir)
 
+    metadata = {'tool': 'checkm2',
+                'version': '1.0.1',
```

Comment:
It seems like it'd be unlikely for this to be updated when the tool version in the Dockerfile is updated. Maybe it should go in a file somewhere that both places read?

Reply:
added reminder comments in docker file and versions.yaml

Reply:
:+1: